compiler/luci/pass/include/luci/CircleOptimizer.h

```diff
@@ -67,6 +67,7 @@ class CircleOptimizer final
       SubstitutePadV2ToPad,
       SubstituteSqueezeToReshape,
       ConvertNCHWToNHWC,
+      RemoveBroadcast,
```

Comment:
As this is about `Const` broadcast, can we use `RemoveConstBroadcast`?
Um... purpose is to remove broadcasting in successor, actual thing is to expand Const to match shape...
`RemoveConstBroadcast` itself seems not quite fit...
@jinevening , can you propose a good name for this?

Reply:
> can you propose a good name for this?

I think `RemoveConstBroadcast` or `ExpandBroadcastConst` is a better name.

Plus, I think that it would be better to implement this transformation in the backend.

Reply:
The transformation code looks ok to me.

Reply:
yes model size may grow... as this is enabled by option switch we can provide in frontend:)

Reply:
@ai-moiseev if possible could you consider the pass name modification?

Reply:
Done, I've taken ExpandBroadcastConst variant, since no removal takes place here, but rather expansion.